src/config/entities/__tests__/configuration.ts

```diff
@@ -135,8 +135,8 @@ export default (): ReturnType<typeof configuration> => ({
     },
   },
   redis: {
-    host: faker.internet.domainName(),
-    port: faker.internet.port().toString(),
+    host: process.env.REDIS_HOST || 'localhost',
+    port: process.env.REDIS_PORT || '6379',
```

Comment:
The description of the PR mentions a test instance but if we follow the setup that we have with `docker-compose` then the development instance will be used.

While we use Redis as a key-value store to temporarily store data, if you run the tests after a debug session for example, you might get the wrong data (and assertions might fail).

Reply:
This is in part now mitigated with https://github.com/safe-global/safe-client-gateway/pull/998

Reply:
Good point, I also thought about this, and I was thinking of using a separate Redis _database_ (logic data separation) between test data and _development_ data. I decided against doing that in this PR to avoid introducing too much noise, but we can easily set `database: 1` when running tests (if not specified, all the data is stored in database 0 in Redis). Context: https://redis.io/commands/select/

What do you think about this solution? It wouldn't overlap with https://github.com/safe-global/safe-client-gateway/pull/998, as running tests in a separate database doesn't help with parallel-running test collisions.

Reply:
I'm more in favour of a separate Redis instance for the tests. The database selection would require more changes on the production code to have a default one selected.

One advantage of having a separate instance is also performance – running the tests while running a development build would not affect the underlying performance (even with different databases).

Although this is not as critical as a separate database for testing due to the nature of the data that we store in Redis.

Reply:
> The database selection would require more changes on the production code to have a default one selected.
> 
> One advantage of having a separate instance is also performance – running the tests while running a development build would not affect the underlying performance (even with different databases).
> 

I totally agree with the performance part, even if it's not critical I'd say. Regarding the changes to production code, I think the tests involving Redis are using a test  `RedisClient`, provided by `redisClientFactory` (under the `__tests__` folder), so the database selection (via parameter, or even hardcoding the database number) would affect test classes only if I'm not wrong.

But yeah, overall I also think that having a separate instance makes more sense, and it's more consistent with the approach we took for the Postgres database also 👍🏻